Mathlib/AlgebraicTopology/SimplexCategory/Defs.lean

```diff
@@ -189,6 +193,29 @@ scoped macro_rules
   | `(⦋$m:term, $p:term⦌$n:subscript) =>
     `((⟨SimplexCategory.mk $m, $p⟩ : SimplexCategory.Truncated $n))
 
+/-- Make a morphism in `Truncated n` from a morphism in `SimplexCategory`. This
+is equivalent to `@id (⦋a⦌ₙ ⟶ ⦋b⦌ₙ) f`. -/
+abbrev Hom.tr {n : ℕ} {a b : SimplexCategory} (f : a ⟶ b)
+    (ha : a.len ≤ n := by trunc) (hb : b.len ≤ n := by trunc) :
+    (⟨a, ha⟩ : Truncated n) ⟶ ⟨b, hb⟩ :=
+  f
+
+lemma Hom.tr_comp {n : ℕ} {a b c : SimplexCategory} (f : a ⟶ b) (g : b ⟶ c)
+    (ha : a.len ≤ n := by trunc) (hb : b.len ≤ n := by trunc)
+    (hc : c.len ≤ n := by trunc) :
+    tr (f ≫ g) = tr f ≫ tr g :=
+  rfl
+
+/-- The inclusion of `Truncated n` into `Truncated m` when `n ≤ m`. -/
+def incl (n m : ℕ) (h : n ≤ m := by omega) : Truncated n ⥤ Truncated m where
+  obj a := ⟨a.1, a.2.trans h⟩
+  map := id
+
+/-- For all `n ≤ m`, `inclusion n` factors through `Truncated m`. -/
+lemma incl_comp_inclusion {n m : ℕ} (h : n ≤ m) :
```

Comment:
These lemmas (as well as `truncation_comp_trunc`) do not seem to be used yet. Instead (or in addition to these), could you add the natural isos versions of these lemmas?

Reply:
I elected to replace the existing lemmas with natural iso versions of the same name. Maybe we want to name these `incl_comp_inclusion_iso` in case we later decide we also want the equalities, though?